@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgSounds.hpp

```diff
@@ -136,4 +136,18 @@ class CfgSounds
 	};
 	///END OF DOOR SOUNDS///
 
+	///ETL AND VRS SOUNDS///
+	class ETL_VRS_Shake_01
+	{
+	    sound[] = {"\fza_ah64_us\audio\CreakingAirFrame.ogg", 1, 1};
+	    frequency = "CustomSoundController3";
+	    volume = "(1-camPos)*rotorSpeed*CustomSoundController4";
+	};
+	class ETL_VRS_Shake_02
+	{
+	    sound[] = {"A3\Sounds_F\vehicles\noises\vehicle_stress3", 1, 1};
+	    frequency = "CustomSoundController3";
+	    volume = "(1-camPos)*rotorSpeed*CustomSoundController4";
```

Comment:
Do these work as set here? And how are they set to only work for the Apache?

Reply:
Moved to their correct location